draft-ietf-httpbis-cache-digest.md

```diff
@@ -286,6 +287,37 @@ Additionally, User Agents SHOULD NOT send CACHE_DIGEST when in "privacy mode."
 
 --- back
 
+# Encoding the CACHE_DIGEST frame as an HTTP Header
+
+Some HTTP/2 protocol stacks do not provide an interface to inject arbitrary HTTP/2 frames while allowing the application to send additional HTTP request headers.
```

Comment:
Isn't the motivation simply that browsers currently don't send CACHE_DIGEST, so you have to fake it with Fetch?